build-tests/heft-node-basic-test/config/heft.json

```diff
@@ -0,0 +1,11 @@
+{
+  "eventActions": [
+    {
+      "actionKind": "deleteGlobs",
+      "heftEvent": "clean",
+      "actionId": "defaultClean",
+      "globsToDelete": ["dist", "lib", "temp"]
+    }
+  ],
+  "$schema": "https://developer.microsoft.com/json-schemas/heft/heft.schema.json"
```

Comment:
For the `basic-test` projects, could you use the full file template with the comments?  The tutorials refer people to these projects as examples, and the comments are definitely helpful for real world projects.

Also `"$schema"` is supposed to be the first thing, not the last thing, in the file.

Reply:
I wrote a tool to generate these files. I figured we'd replace this pretty quickly with a rig.

Reply:
Cleaned these up.